packages/pangea-sdk/.sdk-ci.yml

```diff
@@ -35,6 +35,13 @@ generate_docs:
     paths: ['packages/pangea-sdk/python_sdk.json']
     when: on_success
 
+mypy:
+  extends: .pangea-sdk-base
+  stage: lint
+  needs: []
+  script:
+    - poetry run mypy pangea tests
```

Comment:
what's the image being used in tests?

Reply:
Should be `python:3.7-bullseye` (from root `.gitlab-ci.yml`).